lib/ui/pages/profile/update_profile/update_profile_page.dart

```diff
@@ -52,9 +52,8 @@ class _UpdateProfileChildPageState extends State<UpdateProfileChildPage>
     final user = context.read<UserCubit>().state.user;
     textNameController = TextEditingController(text: user?.username ?? '');
     textBirthdayController = TextEditingController(
-        text: DateTimeExtension(user?.birthday ?? DateTime.now())
-            .toDateString()
-            .toString());
+      text: AppDateUtils.toDateString(user?.birthday ?? DateTime.now()),
```

Comment:
Logic sai
Nếu user chưa set Date of birth thì phần UI hiển thị placeholder text